### PR TITLE
Slovak Update monero-core_sk.ts

### PR DESCRIPTION
**Pull request by jarole**
sk lang. for 0.13

reference to (#1582)

### CODE REVIEW

File: translations/monero-core_sk.ts [hunk 1]
```diff
@@ -2623,82 +2632,82 @@ Poplatok: </translation>
...
-        <translation type="unfinished"></translation>
+        <translation>VAROVANIE: neštandardná veľkosť kruhu, môže narušiť súkromie. Predvolená a odporúčaná veľkosť je 7 </translation>
     </message>
     <message>
         <location filename="../main.qml" line="640"/>
         <source>
 
 Number of transactions: </source>
-        <translation type="unfinished"></translation>
+        <translation>Počet transakcií</translation>
```
Comment (erciccione) [inline translations/monero-core_sk.ts]: missing `:` + whitespace

File: translations/monero-core_sk.ts [hunk 2]
```diff
@@ -2623,82 +2632,82 @@ Poplatok: </translation>
...
 Description: </source>
-        <translation type="unfinished"></translation>
+        <translation>Popis: </translation>
     </message>
     <message>
         <location filename="../main.qml" line="643"/>
         <source>
 Spending address index: </source>
-        <translation type="unfinished"></translation>
+        <translation>index míňacej adresy (spending)</translation>
```
Comment (erciccione) [inline translations/monero-core_sk.ts]: missing `:` + whitespace

**Comment by erciccione** [comment]
Good, can you squash those commits into just one? If you don't ue git you can add me as a collaborator to your repo and i will fix those for you

**Comment by jarole** [comment]
Sorry I am newbie here, do you mean squash using git command line utility (not familiar yet) or do you mean squash button somewhere on this site which I cannot find?

I have sent you an invitation to my repo already
https://github.com/jarole/monero-gui/invitations

In case you have a lot of time, please describe how to do it. so I can be more agile in the future.

**Comment by jarole** [comment]
missclick and close by mistake

**Comment by erciccione** [comment]
i meant using the command line, bu anyway, i saw the invitation and already squashed. As you can see there is only one commit now.

> In case you have a lot of time, please describe how to do it. so I can be more agile in the future.

I don't think i will have time until the release is out, but we can work on that together for your next contribution. Thank you for this one BTW :)

**Pull request merged**